DESCRIPTION

```diff
@@ -78,4 +78,4 @@ Language: en-US
 LazyData: true
 Roxygen: list(markdown = TRUE, roclets = c("collate", "namespace",
     "admiraldev::rdx_roclet"))
-RoxygenNote: 7.3.2
+RoxygenNote: 7.3.2.9000
```

Comment:
@jimrothstein the roxygen version should not change.  also noticed that 128 `.Rd` files changed perhaps that is due to this.  some `.Rd` files should change, but just not that many :)

Reply:
yikes ... `think` fixed
- now using roxygen2 7.3.2
- sync to github (git push,  then git pull ... all branches)